README.md

```diff
@@ -249,6 +249,7 @@ Inspired by [awesome-php](https://github.com/ziadoz/awesome-php).
         * [awesome-flake8-extensions](https://github.com/DmytroLitvinov/awesome-flake8-extensions)
     * [pylama](https://github.com/klen/pylama) - A code audit tool for Python and JavaScript.
     * [pylint](https://www.pylint.org/) - A fully customizable source code analyzer.
+    * [ruff](https://github.com/charliermarsh/ruff) - An extremely fast Python linter, written in Rust.
```

Comment:
Hello @peidrao,
Starting from version [0.1.2](https://github.com/astral-sh/ruff/releases/tag/v0.1.2), `ruff` is not only a linter, but also [a code formatter](https://github.com/astral-sh/ruff/pull/7732/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5L13-R13).
```suggestion
    * [ruff](https://github.com/charliermarsh/ruff) - An extremely fast Python linter and code formatter, written in Rust.
```